JavaScript/Topics/Popular Libraries/Tailwind CSS/README.md

```diff
@@ -0,0 +1,119 @@
+
+<img src="img 1 Hello World.png">
+
+## Adding Tailwind to your CSS:-
+
+Now that you have installed Tailwind CSS using NPM, you will need to use the custom ```@tailwind``` directive and inject Tailwind’s base, components, and utilities styles into your main CSS file.
+
+To do this, create a new CSS file called main.css (or any other preferred name) and add the following lines of code inside it:
+
+```html
+
+@tailwind base;
+
+@tailwind components;
+
+@tailwind utilities;
+```
+
+What this does is that as soon as this file will be processed using Tailwind it will be swapped with the actual CSS styles from Tailwind. You’ll see soon exactly what will happen.
+
+## Creating the configuration file and process your CSS with Tailwind:-
+
+The recommended way of developing with Tailwind CSS is to have a configuration file that you will edit along the way based on your needs. You will be able to easily add and modify the default utility classes from Tailwind. Practically, it will be very easy to change fonts, colors, sizings, spacings and literally anything.
+
+Using the Tailwind CLI run the following command and generate a boilerplate configuration file:
+
+```npx tailwindcss init```
+
+This will generate a file called tailwind.config.js having the following content:
+
+```js
+// tailwind.config.js
+module.exports = {
+    future: {},
+    purge: [],
+    theme: {
+    extend: {},
+    },
+    variants: {},
+    plugins: [],
+}
+```
+
+Now add the following two plugins:
+
+```plugins
+
+plugins: [
+    // ...
+    require('tailwindcss'),
+    require('autoprefixer'),
+    // ...
+]
+```
+
+## Processing your CSS with Tailwind:-
+
+Later in this tutorial I will show you how to automatically watch for changes in your CSS file, but for now you can just run the following command to compile the main.css file and create a new file called output.css, which will be the actual CSS file that will be included in the HTML files:
+
+```npx tailwindcss build main.css -o output.css```
+
+If you take a look at your folder structure, you will see that a new output.css file was generated which holds all of the default Tailwind CSS utility classes based on the configuration file that we have created.
+
+## Including Tailwind CSS into your HTML templates:-
+
+Create an HTML file called index.html in your root project directory and make sure to include the output.css CSS file inside the head tag of your template:
+
+```html
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tailwind CSS Tutorial by Themesberg</title>
+    <link rel="stylesheet" href="output.css">
+</head>
+<body>
+    <!-- Build awesome user interfaces using Tailwind CSS starting from here -->
+</body>
+</html>
+```
+
+_Congratulations! You have successfully set up Tailwind CSS using the recommended method by the official documentation._
+
```

Comment:
Hey @SakshiMishra1 awesome work 🙂. Just to elaborate could you also includes Merits and Demerits of using Tailwind CSS here.